IRCBot/index.ts

```diff
@@ -0,0 +1,31 @@
+import { BanchoClient } from "bancho.js";
+import { config } from "node-config-ts";
+import { handleCommand } from "./commands";
+
+const banchoClient = new BanchoClient({ username: config.osu.irc.username, password: config.osu.irc.ircPassword, botAccount: config.osu.irc.botAccount });
+
+async function main() {
+    await banchoClient.connect();
+
+    const self = banchoClient.getSelf();
+    console.log(`osu! IRC Bot connected as ${self.ircUsername}`);
+
+    banchoClient.on("PM", async (message) => {
```

Comment:
Also add an event check for https://bancho.js.org/BanchoMultiplayerChannel.html#event:message for ref setup, could might as well have it share the same commands too

Reply:
pls see https://github.com/Corsace/Corsace/pull/208/commits/c22ce32173b4b62076947937cd6bd0091d1c4244